### ISSUE TITLE
SUGGESTION DRAGULA / LA MAS DRAGA 

### ISSUE THREAD

**Issue by sschery** [MyRainboww/Drag-Race-Simulator#21]
First of all, I love this Simulator so much and I play with it all the time!

Would it be possible to have the Dragula and La Más Draga seasons in the simulation? I think that would be iconic mixing in different queens/ ghouls. 

Thank you for being excellent and upgrading this iconic Simulator. There's no hurry because you already update it frequently.

**Comment by MyRainboww** [comment]
Hello! Thank you so much! I'm so happy that you are enjoying it <3

I don't know about adding them right now, the reason is be cause both Dragula and La Más Draga are mostly runway-based competitions. I would love to add them but it would be hard to think about their stats for Drag Race Challenges.
What do you think?

**Comment by sschery** [comment]
I agree! I know challenges-wise: you could incorporate instead of the standard "girl groups" they use rock bands for Dragula. They use multiple bttm placements (bttm 2, 3, 4). Include the floorshow format as the runway and for the exterminations "EXT", you can come up with gruesome challenges (Fear Factor for example) or use what the show has used.  They use Glamour, Filth, and Horror format in the finale for the finale, so maybe incorporate that. 
For La Más Draga, you can use the standard simulation layout as normal. - they use the same format as the normal seasons of Drag Race, lip-synch for your life. 
- Also I thought of including a Top 2 whenever (Camden and Daya for example)
- The cast of Drag Race España S2
That's what I can think of at the top of my head. Thank you again for such an amazing game! :)

**Comment by ThawedMist** [comment]
> Hello! Thank you so much! I'm so happy that you are enjoying it <3
> 
> I don't know about adding them right now, the reason is be cause both Dragula and La Más Draga are mostly runway-based competitions. I would love to add them but it would be hard to think about their stats for Drag Race Challenges. What do you think?

Don't mean to jump onto this but here are some challenges i know they do. 
Design, Acting, Performance for judges (not sure what this is called.)  Presenting looks (ball?) Photoshoot, Girl group (Band) 
Sources: https://rupaulsdragrace.fandom.com/wiki/The_Boulet_Brothers%27_Dragula_(Season_4) (seasons have ideas on what challenges are, you could probably just do a bunch of designs with descriptions both from there and made up, like "Cowboy Floorshow" or "Killer Clown Floorshow"
